src/pages/audio-rewards-page/store/sagas.ts

```diff
@@ -186,6 +185,7 @@ function* claimChallengeRewardAsync(
           default:
             throw new Error(`Unknown Error: ${response.error}`)
         }
+        yield put(claimChallengeRewardWaitForRetry(claim))
```

Comment:
Moved this to the bottom of the switch to prevent a "bug" where unknown errors would still put a retry in the store